static/img/blog/phase-3-timeline/test

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can we delete this file?

Reply:
yes - I want to start grouping images for specific blog posts in folders, but (due surely to user error) I can't seem to create a folder in GitHub without creating a dummy file in a folder and then uploading my images. Sorry for the confusion.

Reply:
Ah, gotcha. Yeah, Github won't let you add empty folders, so definitely not user error. Using a dummy file is totally fine, I just wan't sure why it was there.